components/controls/Select.css

```diff
@@ -339,7 +339,8 @@
 
 .Select-option.is-disabled {
   cursor: default;
-  opacity: 0.5;
+  opacity: 0.4;
+  font-style: italic;
```

Comment:
.5 opacity already results in a failure in contrast rating, so I don't think it's a good idea to make it even more faded.

Reply:
Laura suggested this change, because currently the contrast is too high on those disabled items and they don't seem like being disabled.
In any case I think we won't ever meet a correct contrast rating on that kind of disabled stuff, so it seems ok to me to lower it a bit more to make it more obvious that the item is disabled.